src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,38 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.10.0
+5 July 2023
+
+## Removal Notice
+
+Kubernetes 1.23 is no longer supported.
+
+### Features
+
+- Now you can select an expiration period when creating a new Personal Access Token. <!-- 6690 -->
+- Administrators have the ability to wake up, put to sleep, or mark any preview environment as keep-awake. <!-- 6662, 6622 -->
+- Upgraded ingress-nginx to [4.6](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.7.0). <!-- 6625 -->
+
+### Improvements
+
+- Kubernetes Annotations can now be added to: <!-- 6689, 6668 -->
+  - Default self-signed wildcard certificate via `.Values.wildcardCertificate.annotations`.
+  - Internal self-signed webhook certificate via `.Values.webhook.internalCertificate.annotations`.
+  - Dockerconfig via `.Values.secret.annotations`.
+- Kubernetes Labels can now be added to: <!-- 6668 -->
+  - Dockerconfig via `.Values.secret.labels`.
+
+### Bugfixes
+
+- Fixed an installation step warning that was displayed incorrectly when using the default `.Values.subdomain` (`localtest.me`). <!-- 6717 -->
```

Comment:
What was the specific warning? It could be a "nice to have" to add a bit more specificity here, but non-blocking